### PR TITLE
fix: add 24h read/write metrics to d1 info

### PR DESCRIPTION
**Pull request by rozenmd**
output for experimental dbs:
```
┌───────────────────┬──────────────────────────────────────┐
│                   │ xxxx-xxxx-xxxx-xxxx-xxxx             │
├───────────────────┼──────────────────────────────────────┤
│ name              │ northwind                            │
├───────────────────┼──────────────────────────────────────┤
│ version           │ beta                                 │
├───────────────────┼──────────────────────────────────────┤
│ num_tables        │ 13                                   │
├───────────────────┼──────────────────────────────────────┤
│ file_size         │ 33.1 MB                              │
├───────────────────┼──────────────────────────────────────┤
│ running_in_region │ WEUR                                 │
├───────────────────┼──────────────────────────────────────┤
│ read_queries_24h  │ 2219                                 │
├───────────────────┼──────────────────────────────────────┤
│ write_queries_24h │ 0                                    │
└───────────────────┴──────────────────────────────────────┘
```

```
{
  "uuid": "xxxx-xxxx-xxxx-xxxx-xxxx",
  "name": "northwind",
  "version": "beta",
  "num_tables": 13,
  "file_size": 33067008,
  "running_in_region": "WEUR",
  "read_queries_24h": 2219,
  "write_queries_24h": 0
}
```

### CODE REVIEW

**Review by elithrar — CHANGES_REQUESTED** [review]
@rozenmd - should we change `file_size` to something clearer - e.g. `database_size` - ?

Since `file_size` implies a single file/isn't a term we use elsewhere when talking about D1.

Otherwise: LGTM!

File: packages/wrangler/src/d1/info.tsx [hunk 1]
```diff
@@ -45,16 +47,88 @@ export const Handler = withConfig<HandlerOptions>(
...
 		);
+
+		let output: Record<string, string | number> = { ...result };
+		if (output["file_size"]) {
+			output["database_size"] = output["file_size"];
+			delete output["file_size"];
+		}
+		if (result.version === "beta") {
+			const today = new Date();
+			const yesterday = subDays(today, 1);
```
Comment (petebacondarwin) [inline packages/wrangler/src/d1/info.tsx:58]: This is overkill to install and use the date-fns library just for this `subDays()` function. In this context it is equivalent to:

```
yesterday = new Date();
yesterday.setDate(yesterday.getDate() - 1);
```

See https://github.com/date-fns/date-fns/blob/e36a05cea15fda926ea93108f661e0aed4a4bed9/src/addDays/index.ts#L31

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: packages/wrangler/package.json [hunk 2]
```diff
@@ -100,6 +100,7 @@
 		"@esbuild-plugins/node-modules-polyfill": "^0.1.4",
 		"blake3-wasm": "^2.1.5",
 		"chokidar": "^3.5.3",
+		"date-fns": "^2.30.0",
```
[the remark this answers is not in the document]
Comment (JacobMGEvans) [reply packages/wrangler/package.json:103]: If we were to add anything I would want to add something that would be native eventually, like Temporal API 

https://github.com/js-temporal/temporal-polyfill

File: packages/wrangler/src/d1/info.tsx [hunk 3]
```diff
@@ -45,16 +47,88 @@ export const Handler = withConfig<HandlerOptions>(
...
+				}
+			);
+			output = {
+				...output,
+				...(graphqlResult && {
+					read_queries_24h: metrics.readQueries,
+					write_queries_24h: metrics.writeQueries,
+				}),
+			};
+		}
```
[the remark this answers is not in the document]
Comment (rozenmd) [reply packages/wrangler/src/d1/info.tsx:110]: Good point, yeah

File: packages/wrangler/src/d1/info.tsx (hunk 1 above)
Comment (rozenmd) [reply packages/wrangler/src/d1/info.tsx:58]: It's more like:
```
const today = new Date();
const yesterday = new Date(new Date(today).setDate(today.getDate() - 1));
```

But sure, can do

**Pull request merged**